google-chart.html

```diff
@@ -384,6 +404,7 @@
             var chart = chartAndData[0];
             var data = chartAndData[1];
             google.visualization.events.addOneTimeListener(chart, 'ready', function() {
+              this._setImageUri(this.imageURI);
```

Comment:
Hmmm... I tried a this suggestion as well as a few variations on it, but I couldn't get the upward binding to work. Is a property with a leading `_` special in some way? I haven't seen anything like this.